Intersect.Client/Interface/Game/EntityPanel/EntityBox.cs

```diff
@@ -109,7 +124,7 @@ public EntityBox(Canvas gameCanvas, EntityType entityType, Entity myEntity, bool
             EntityType = entityType;
             PlayerBox = playerBox;
             EntityWindow =
-                playerBox ? new ImagePanel(gameCanvas, "PlayerBox") : new ImagePanel(gameCanvas, "TargetBox");
+                PlayerBox ? new ImagePanel(gameCanvas, "PlayerBox") : new ImagePanel(gameCanvas, "TargetBox");
```

Comment:
lets rename this variable to isPlayerbox, what do you think?

Reply:
Sure 👍🏽

Reply:
It should be a private boolean as well, it's never used outside this scope